transform/transformers/IndexFile.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+import dateutil.parser
+
+from io import BytesIO
+from transform import settings
+from transform.views.image_filters import get_env, format_date
+
+
+class IndexFile:
+    """Class for creating in memory index_file file using StringIO."""
+
```

Comment:
Says StringIO in docs but you're actually using BytesIO

Reply:
Thanks , good catch , I had changed code and not doc strings